README.rst

```diff
@@ -2,11 +2,17 @@
  z3c.bcrypt
 ============
 
+.. warning:: z3c.bcrypt has been superseeded with the new version of
+            `zope.password`_ as it now includes `bcrypt` support based on
+            the well-maintained `bcrypt`_ library. Please do no longer use
```

Comment:
"Please do no longer" is an awkward phrase. Can it just be "Please do not"?

Reply:
fixed